CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Upgrade MSAL to `4.59.1`.
```

Comment:
[small nit]: Do you want to mention the current version too? Something like `Upgrade MSAL from 4.55.0 to 4.59.1`. Might help us when we quickly want to look at the MSAL changelog if there are any bugs.

Reply:
Done